ansible/roles/test/tasks/everflow_testbed/apply_config.yml

```diff
@@ -4,47 +4,16 @@
     docker_testdir: /tmp
     tests_location: roles/test/tasks/everflow_testbed
     testname: apply_config
-    config_files:
-        - session.json
-        - acl_table.json
-        - acl_rule_persistent.json
 
-- name: Generate json files with ACL configuration for tests
-  template: src={{ tests_location }}/{{ testname }}/acl_table.j2 dest={{ tests_location }}/{{ testname }}/acl_table.json
-  connection: local
+- name: Get session info.
+  include: roles/test/tasks/everflow_testbed/get_session_info.yml
 
-- name: Initialize config test
-  include: roles/test/tasks/init_config_test.yml
```

Comment:
Do we still need init_config_test.yml and run_config_test.yml files?

Reply:
Currently we still using them in acl/everflow dev tests. Will remove them during adaptation of dev tests.